_docs/usecases/docker/cassandra.md

```diff
@@ -0,0 +1,111 @@
+---
+layout: guide
+title: StorageOS Docs - Cassandra
+anchor: usecases
+module: usecases/docker/cassandra
+---
+
+# ![image](/images/docs/explore/cassandralogo.png) Cassandra with StorageOS:
+
+Cassandra is a popular distributed NoSQL open source database. 
+
+## Cassandra and StorageOS
+
+There are several benefits with deploying Cassandra with StorageOS:
+
+* Instant, stateless Cassandra application containers on demand
+* Persistent, highly available storage for application and database data
+
+Before you start, ensure you have StorageOS installed and ready on a Linux
+cluster.
+
+## Create a Cassandra Volume
+
+1. Create a 1GB volume called `cassandra-data` in the default namespace for
+   each Cassandra ring member. In this example we will have a three member ring so
+   three volumes will be created. 
+
+   ```bash
+   $ docker volume create --driver storageos --opt size=1 cassandra-data
+   cassandra-data
+   $ docker volume create --driver storageos --opt size=1 cassandra-data2
+   cassandra-data
+   $ docker volume create --driver storageos --opt size=1 cassandra-data3
+   cassandra-data
+   
+   $ docker volume list
+   DRIVER              VOLUME NAME
+   storageos           cassandra-data
+   storageos           cassandra-data2
+   storageos           cassandra-data3
+   ```
+
+1. Run a Cassandra container using the StorageOS volume driver.
+
+   ```bash
+   $ docker run -d --name cassandra-docker \
+   -v cassandra-data:/var/lib/cassandra    \
+   cassandra:3.11
+   e701e4945fb6730127ecfce8459faad5d6a08fbd53dd6f68f336ff8bf092d470
```

Comment:
What's the purpose of this GUID here?

Reply:
Just an example of the output you'd expect from the command. I can see how it's a little misleading as others won't get that GUID. I can remove it.

Reply:
It's been removed